Automattic/sensei
# Fix learning mode lesson button, image and featured video style in Course theme

Resolves https://github.com/Automattic/sensei/issues/7134

## Proposed Changes
* The buttons under the lessons in Learning Mode looked narrower than the design. So we've changed the padding and fixed some other discrepancies like gap, text spacings etc to make it match the design.
* I didn't find any design issue with the featured image for lesson. If we add normal images which are not featured image, they don't have the border and border radius. So we've added border with radius around them.
* Also, if we add featured video, it didn't have border with radius around it. So we've added that as well.

## Testing Instructions


1. Activate Course theme
2. Select the gold variant of the theme
3. Create a course, make sure LM is enabled for that course
4. Create a lesson in that course with a quiz
5. Add a featured image on top of that lesson
6. Add a normal image somewhere inside that lesson
7. Now take that course as a Student and view that lesson
8. Check the buttons below
9. Make sure all of those match the design
10. Match the design for other variations of Course theme
11. Replace the featured image with a featured video and follow the steps above again.

### Before

![image](https://github.com/Automattic/sensei/assets/6820724/984d403c-c090-46b9-a0cf-9f82892200b5)

### After

![image](https://github.com/Automattic/sensei/assets/6820724/2662fb81-1ac4-4430-adc8-dded88a714e8)

## Pre-Merge Checklist
